javascript/javascript2/week3/FastetsPresser/index.js

```diff
@@ -0,0 +1,111 @@
+// Confetti function
+const canvas = document.querySelector("#my-confetti");
+const jsConfetti = new JSConfetti();
+
+let timerStarter;
+let timerStoped = true;
+let clInterval;
+let clInterval2;
```

Comment:
If you define these variables within the scope of the functions instead of here, there is no need for an interval 1 or 2. You can just do the following in both the timer() and countDownBeforeStart() function:

```
const interval = setInterval(tick, 1000);
....
clearInterval(interval);
```

Also, I'd recommend to not make variable names shorter. It's better to just write `clearInterval` instead of `clInterval`, since it's more readable and your future fellow programmers will understand it much faster :)

Reply:
I decided to make it shorter because the name of function that stops timer is also clearInterval. So it will be like clearInterval(clearInterval)